src/aap_eda/api/views/user.py

```diff
@@ -249,5 +250,11 @@ def destroy(self, request, *args, **kwargs):
         if instance == request.user:
             return Response(status=status.HTTP_403_FORBIDDEN)
 
+        if settings.DIRECT_SHARED_RESOURCE_MANAGEMENT_ENABLED:
```

Comment:
@bzwei can we use a better name for this setting maybe

CLONED_REMOTE_RESOURCE_ENABLED

or
USE_REMOTE_RESOURCES

Maybe we have a list of objects that are remote resources so if we need to add something down the road like Groups etc we can just add it to that list.

Reply:
+1 for `USE_REMOTE_RESOURCES`

Reply:
This term is specified in the jira issue, used by all products across AAP.